tensorflow/python/kernel_tests/zero_division_test.py

```diff
@@ -54,11 +54,7 @@ def testZeros(self):
             #
             # XLA constant folds integer division by zero to 1.
             self.assertTrue(test.is_gpu_available())
-            if not test.is_built_with_rocm():
-              # division by zero yields a different pattern on AMD GPUs
-              # TODO(rocm) : investigate whether the resulting bit pattern on
-              # AMD GPUs is deterministic
-              self.assertIn(result, (-1, 1, 0xff, 0xffffffff))
+            self.assertIn(result, (-1, 1, 2, 0xff, 0xffffffff))
```

Comment:
thanks again for looking into this.
wonder what changed on our end to make this pass now.

Reply:
I don't know if anything changed. It was failing because division produced 2 in one of the sub-cases (int64) and 2 was not in the list.